tests/test_integration_partition.py

```diff
@@ -0,0 +1,48 @@
+from indextools.console import partition
+from pathlib import Path
+import filecmp
+import pytest
+
+INPUT_DATA_DIR = Path(__file__, "..", "input_data").resolve()
```

Comment:
Take a look at https://pypi.org/project/pytest-datadir-ng/. I think this is the right way to access data directories.